src/PostContent/PostContent.tsx

```
@@ -2,10 +2,10 @@ import { Typography } from "hazel-ui/Typography";
 import { MDXRemote } from "next-mdx-remote";
 import Head from "next/head";
 
-import type { Post } from "../lib/posts.js";
-import { Date } from "../ui/Atoms/Date.js";
-import { Layout } from "../wrappers/Layout.js";
-import { config } from "../config.js";
+import type { Post } from "../lib/posts";
+import { Date } from "../ui/Atoms/Date";
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)